README.md

```diff
@@ -171,6 +171,20 @@ And finally (do not forget):
 
 		LC_ALL=C repoowner=theforeman branch=master proxy_repo=1.14 vagrant destroy $distro
 
+Extensions
+----------
+
+Discovery Image supports runtime extensions published via TFTP or HTTP.
+Those are distributed as ZIP file with shell scripts. It is also possible
```

Comment:
Those are distributed as ZIP files with shell scripts. It is also possible to build an image with extensions built-in which is helpful for PXE-less environments.